### PR TITLE
Use the QTensor with QReLU

### PR DESCRIPTION
**Pull request by z-a-f**
Stack:
&nbsp;&nbsp;&nbsp;&nbsp;:black_circle:&nbsp; **#19312 Use the QTensor with QReLU**&nbsp;&nbsp;[:green_heart:](https://our.internmc.facebook.com/intern/diff/D14819460/)

Replaces the tuple hack with the QTensor. Please, note this can be landed ONLY after #18960 (D14810261) is landed.

Differential Revision: [D14819460](https://our.internmc.facebook.com/intern/diff/D14819460/)

### CODE REVIEW

File: aten/src/ATen/native/quantized/cpu/qrelu.cpp [hunk 1]
```diff
@@ -1,44 +1,39 @@
...
         "",
-        std::vector<c10::Argument>{c10::Argument("x", TensorType::get()),
-                                   c10::Argument("scale", FloatType::get()),
-                                   c10::Argument("zero_point", IntType::get())},
-        std::vector<c10::Argument>{
-          c10::Argument("y", TensorType::get()),
-          c10::Argument("scale_o", FloatType::get()),
-          c10::Argument("zero_point_o", IntType::get())
-        }),
+        std::vector<c10::Argument>{c10::Argument("qx", TensorType::get())},
```
Comment (raghuramank100) [inline aten/src/ATen/native/quantized/cpu/qrelu.cpp]: Lets have the name be aten::quantized_relu

File: aten/src/ATen/native/quantized/cpu/qrelu.cpp [hunk 2]
```diff
@@ -1,44 +1,39 @@
...
-    return std::make_tuple(tensor, scale, zero_point);
+  QTensor operator()(const at::QTensor& qx) {
+    QTensor qy = at::empty_affine_quantized(qx.sizes(),
+                                            at::device(kCPU).dtype(kQInt8),
+                                            qx.q_scale().toDouble(),
+                                            qx.q_zero_point().toLong());
+    auto iter = TensorIterator::unary_op(qy, qx);
+    const auto zero_point = qx.q_zero_point().toByte();
+    unary_kernel(*iter, [&](c10::qint8 value) -> c10::qint8 {
+      return c10::qint8(std::max(value.val_, zero_point));
```
Comment (raghuramank100) [inline aten/src/ATen/native/quantized/cpu/qrelu.cpp]: Please replace with an avx call in a later PR.

Comment (zafartahirov) [reply aten/src/ATen/native/quantized/cpu/qrelu.cpp]: Will do in the upcoming diffs

**Comment by zafartahirov** [comment]
The tests will continue failing until #19714 is landed (D15077095)

**Pull request closed without merging**